system/global.nix

```diff
@@ -47,6 +47,22 @@
     fuse-overlayfs
   ];
 
+  programs.rust-motd = {
+    enable = true;
+    enableMotdInSSHD = true;
+    settings = {
+      banner = {
+        color = "red";
+        command = ''
+          ${pkgs.nettools}/bin/hostname | ${pkgs.figlet}/bin/figlet -f slant
```

Comment:
TIL hostname is not an env variable in bash.

figlet seems cool, but might not survive a rigorous security audit of installed packages. However, then again my user profile is importing bunch of crap if I would import my neovim profile, so for now this is OK.

This is to say, this is one of the reasons I hope to see the nix hash output of proposed changes. That is, I don't know if we already have nodejs in the hosts, or not, but it would be nice to verify to see how much new packages are imported by new changes.

Reply:
Removing fidget, It seems that our hosts have impressively lengthy names

Reply:
Removed figlet, it seems that our hosts have impressively lengthy names